DependencyInjection/Configuration.php

```diff
@@ -651,7 +651,7 @@ public function getConfigTreeBuilder()
                             ->thenInvalid('Service handlers can not have a formatter configured in the bundle, you must reconfigure the service itself instead')
                         ->end()
                         ->validate()
-                            ->ifTrue(function ($v) { return ('fingers_crossed' === $v['type'] || 'buffer' === $v['type'] || 'filter' === $v['type']) && 1 !== count($v['handler']); })
+                            ->ifTrue(function ($v) { return ('fingers_crossed' === $v['type'] || 'buffer' === $v['type'] || 'filter' === $v['type']) && (empty($v['handler']) || !is_string($v['handler'])); })
```

Comment:
shouldn't it be just `empty($v['handler'])`? that would match the other checks around, and match the intent, isn't it?

Reply:
You are right. Changed.